modules/primo/src/Primo/Ting/Result.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * @file
+ * The Result class.
+ */
+
+namespace Primo\Ting;
+
+use Primo\BriefSearch\Document;
+use Ting\Search\TingSearchRequest;
+use Ting\Search\TingSearchResultInterface;
+use TingCollection;
+
+/**
+ * A Ting compatible search result from the Primo search provider.
+ *
+ * @package Primo\Ting
```

Comment:
General question: Is there a reason to keep the `@package` element? I see it as a pre-namespace thing.

Reply:
I agree - It's probably just an IDE thing and it should not be necessary.